src/AbstractFopenFlagOrderFixer.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer;
+
+use PhpCsFixer\Tokenizer\Analyzer\ArgumentsAnalyzer;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @internal
+ *
+ * @author SpacePossum
+ */
+abstract class AbstractFopenFlagOrderFixer extends AbstractFunctionReferenceFixer
```

Comment:
I would suggest removing `Order` from this name, as only one of the extending classes actually has to do with ordering.